### PR TITLE
add more WTD intro stuff to slack welcome message

### PR DESCRIPTION
**Pull request by Bradamant3**
I've been pondering something like this for a while. Several recent events suggest it's overdue. Should have decent time this week to respond to comments.

I wonder whether we want to say something explicit about principles and practices of OSS software somehow also ... but that might be getting too long ...

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
Looks like a 💯 addition.

**Comment by ericholscher** [comment]
I wonder if we can get someone to hack our hubot or slackbot to send this as a welcome message.

**Comment by Bradamant3** [comment]
I looked briefly into hacking the welcome message and didn't see a solution. Which does not mean it doesn't exist -- I'll reach out to the owner of the WomenWhoCodeDC Slack. She does all this stuff brilliantly (that's who I got the annoucements idea from).

**Comment by plaindocs** [comment]
https://api.slack.com/docs/messages maybe?

**Comment by ericholscher** [comment]
Yea, there's a few simple GH solution to it:

https://github.com/slugbay/slack-welcome-bot

But guessing we can hack it into an existing bot we already have.

File: docs/include/slack.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
 
 We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our social rules. All this information is at https://www.writethedocs.org/slack/ -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal!
+
+If you're new to Write the Docs, we also invite you to learn more about our community. Check out how we got started at https://www.writethedocs.org/origin-story/, read who's on the current team at https://www.writethedocs.org/team/, and learn about who we are at https://www.writethedocs.org/about-us/. There's a lot more on the website, too -- explore, get involved, and welcome!
```
Comment (plaindocs) [inline docs/include/slack.txt]: Looks great! 

Unless we can add link syntax I'd be tempted to break this up for readability

* awesome fun text - link
* more fun  - more link

**Review by plaindocs — APPROVED** [review]

File: docs/include/slack.txt (hunk 1 above)
Comment (Bradamant3) [reply docs/include/slack.txt]: Yeah, lazy me had the same thought last night. Fixing up ...

**Comment by plaindocs** [comment]
<3 Danke!

**Comment by plaindocs** [comment]
So I think we can merge this now, but it doesn't really affect much unless we are sending it out.

**Comment by Bradamant3** [comment]
Yeah, this is only part of a larger story. Do we want an issue to track?

**Pull request merged**